pkg/gateway/api/api_grpc.go

```diff
@@ -1567,6 +1563,15 @@ func checkEnvironmentAPIKey(environmentAPIKey *accountproto.EnvironmentAPIKey, r
 	return nil
 }
 
+func contains[T comparable](s []T, e T) bool {
```

Comment:
Go supports generics from 1.18. Because other implementations use the contains function, I unify it using generics.